examples/seismic/acoustic/wavesolver.py

```diff
@@ -211,10 +225,17 @@ def born(self, dmin, src=None, rec=None, u=None, U=None, m=None, **kwargs):
                          save=False, time_order=self.time_order,
                          space_order=self.space_order, dtype=self.model.dtype)
 
+        kwargs.update({'src': src})
+        kwargs.update({'rec': rec})
+        kwargs.update({'u': u})
+        kwargs.update({'U': U})
+        kwargs.update({'m': self.model.m})
```

Comment:
Yeah, I don't understand why this is necessary either.  What's wrong with the previous `op.apply(src=src, rec=rec, u=u...)` approach?